libosmscout-client-qt/include/osmscoutclientqt/MapRenderer.h

```diff
@@ -68,9 +68,9 @@ class OSMSCOUT_CLIENT_QT_API DBRenderJob : public QObject, public DBJob{
 
   void Run(const osmscout::BasemapDatabaseRef& basemapDatabase,
            const std::list<DBInstanceRef> &allDatabases,
-           std::shared_lock<std::shared_mutex> &&locker) override;
+           ReadLock &&locker) override;
 
-  inline bool IsSuccess() const{
+  inline JobStatus  IsSuccess() const{
```

Comment:
Naming is confusing now. But more important is, that there are some places when this enum is assigned to boolean. And assign other value than 0/1 to boolean is undefined-behaviour...

Reply:
All places have been updated to handle enum in the commit. May be I missed one ...

Reply:
`PlaneMapRenderer:403`